chrome/content/zotero/preferences/preferences_export.js

```diff
@@ -133,35 +139,55 @@ Zotero_Preferences.Export = {
 	},
 	
 	
-	updateQuickCopyHTMLCheckbox: function () {
+	updateQuickCopyUI: function () {
 		var format = document.getElementById('zotero-quickCopy-menu').value;
+		
 		var mode, contentType;
 		
-		var checkbox = document.getElementById('zotero-quickCopy-copyAsHTML');
 		[mode, format] = format.split('=');
 		[mode, contentType] = mode.split('/');
 		
+		var checkbox = document.getElementById('zotero-quickCopy-copyAsHTML');
 		checkbox.checked = contentType == 'html';
 		checkbox.disabled = mode != 'bibliography';
+		
+		var menulist = document.getElementById('locale-menu');
+		if (mode != 'bibliography') {
+			// maintain menu selection but show blank label
+			menulist.insertItemAt(0, '', menulist.value);
+			menulist.selectedIndex = 0;
+			menulist.disabled = true;
```

Comment:
> we don't want the label grayed out for styles with default locale (or do we?)

I thought that was the whole point? When a style has a default-locale, that should be the displayed locale, with no option to change it. (since the style locale would override the selected locale anyway)